spec_creation/autofill_eval_spec.py

```diff
@@ -156,9 +156,32 @@ def get_route_from_relation(r):
     return get_coords_for_relation(r["relation_id"],
         r["start_node"], r["end_node"])
 
-def validate_and_fill_leg(orig_leg):
-    # print(t)
+def _add_temporal_ground_truth(orig_loc, start_fmt_date, end_fmt_date):
+    # fill in timespan for which ground truth is valid (see issue #11)
+    # first, if loc is a dict, we need to convert end_loc to a list of dicts.
```

Comment:
nit: "if loc is a dict, we need to convert **loc** to a list of dicts." not `end_loc`